pkg/controller/disruption/disruption.go

```diff
@@ -452,9 +478,21 @@ func (dc *DisruptionController) trySync(pdb *policy.PodDisruptionBudget) error {
 		return err
 	}
 
-	currentHealthy := countHealthyPods(pods)
-	err = dc.updatePdbSpec(pdb, currentHealthy, desiredHealthy, expectedCount)
+	currentTime := time.Now()
+	disruptedPods, recheckTime := buildDisruptedPodMap(pods, pdb, currentTime)
```

Comment:
Can you add a comment? It would be good for us to document these assumptions. (And I assume it's 2 min now.)